tests/integration/test_undrop_query/test.py

```diff
@@ -20,29 +21,25 @@ def started_cluster():
 
 
 def test_undrop_drop_and_undrop_loop(started_cluster):
-    # create, drop, undrop, drop, undrop table 5 times
-    for _ in range(5):
-        table_uuid = str(uuid.uuid1())
-        table = f"test_undrop_loop"
+    uuid_list = []
```

Comment:
The test IMHO runs too long for the stuff it tests. What about this:

- set the drop time setting to 20 sec (XML file).
- create four tables
- drop a table, undrop it, sleep 5 sec (*)
- execute (*) three times in a row
- sleep 10 sec, this should place us ca. 25 sec after test start.
- UNDROP the 4th table, expect that this fails